android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -144,52 +145,55 @@ open class IpnViewModel : ViewModel() {
 
   // Login/Logout
 
+  /**
+   * Order of operations:
+   * 1. editPrefs() with WantRunning=true, LoggedOut=false if AuthKey != null
```

Comment:
nit: maybe something like
```suggestion
   * 1. editPrefs() with maskedPrefs (to allow ControlURL override), WantRunning=true, LoggedOut=false if AuthKey != null
```